deployment-configuration/helm/templates/ingress.yaml

```diff
@@ -38,10 +38,13 @@ metadata:
     cert-manager.io/issuer: {{ printf "%s-%s" "letsencrypt" .Values.namespace }}
     {{- end }}
     nginx.ingress.kubernetes.io/ssl-redirect: {{ (and $tls .Values.ingress.ssl_redirect) | quote }}
-    nginx.ingress.kubernetes.io/proxy-body-size: '250m'
```

Comment:
@filippomc shouldnt we take this 250 for the new situation default value?

Reply:
right that will make us sure it's backwards compatible